### ISSUE TITLE
cannot use zoom functions with screenreader

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-frontend#47]
Testing was done using the JAWS 2019.1810.64 - October 2018 release of Jaws Screen Reading Software.

**Comment by jrynhart** [comment]
Upgrade to UV v3.0.36 seems to have fixed this
Zoom controls contain title attribute
Tested with Chromevox 53 on Windows
